akka-docs/src/main/paradox/typed/cluster.md

```diff
@@ -148,8 +148,8 @@ Please refer to its documentation for more details.
 
 #### Joining configured seed nodes
 
-When a new node is started it sends a message to all seed nodes and then sends join command to the one that
-answers first. If no one of the seed nodes replied (might not be started yet)
+When a new node is started it sends a message to all seed nodes and then sends a join command to the one that
+answers first. If none of the seed nodes replies (might not be started yet)
```

Comment:
Shouldn't that be "If none of the seed nodes reply"?

Reply:
Your question gave me pause because they both sound okay to me, so I went digging a bit.  It seems it can go either way in this case.  Check out this answer on Quora.  (https://www.quora.com/Is-it-more-correct-to-say-None-of-them-are-or-None-of-them-is)  Note this bit:
> When none quantifies a plural noun, both singular and plural agreements are acceptable.

> 
![image](https://user-images.githubusercontent.com/40366664/124501414-595bcd00-dd76-11eb-8660-10266ff70624.png)

I guess we're just in different camps.  :-)  (I'd like to claim that I think of "none of plural noun" as the empty set, and thus singular, but I only came up with that now.  :-)

Reply:
thanks for looking into it, seems we can merge this